ospfd/ospf_vty.c

```diff
@@ -165,7 +181,14 @@ DEFUN_NOSH (router_ospf,
     {
       if (IS_DEBUG_OSPF_EVENT)
         zlog_debug ("Config command 'router ospf %d' received", instance);
-      ospf->oi_running = 1;
+
+      if (ospf->vrf_id != VRF_UNKNOWN)
```

Comment:
Why not just modify the IS_DEBUG_OSPF_EVENT immediately above to have the right information?